src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,153 @@
+# Eye-tracking including gaze position and pupil size
+
+Support for eye-tracking dataset developed as a
+[BIDS Extension Proposal](../extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../introduction.md#citing-bids) on how to
+appropriately credit this extension when referring to it in the context
+of the academic literature.
+
+## Terminology and conventions
+
+Eyetracking-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from other fields.
+This section adds clarifications to
+[Common Principles - Definitions](../common-principles.md#definitions) for the
+eye-tracking context.
+
+-   **Eye-tracker** - refers to the apparatus allowing the records of gaze
+    position and/or pupil size.
```

Comment:
Furthermore, at least one eye tracker (tobii Spectrum) also provides a data stream for the openness of the eye (the distance between the eyelids). Should that be included? Thats changes in many places.

Reply:
ping

Reply:
Given that this is not common to all eyetrackers it may be premature to add this no.
Also nothing prevents users from adding extra "columns" to their data file to store that information.

Reply:
Hmm I'm curious about this, maybe we could add recommended columns in the tabular file... 
But I have never deal with such data.
@dcnieho would you like to join our Matrix channel to share more of this us. Most in general Tobii users could be of great help for this BEP (and you seems to have this experience).
https://matrix.to/#/#bids-bep020:matrix.org